extendeddatacrossword.go

```diff
@@ -56,13 +56,12 @@ func (e *ErrByzantineData) Error() string {
 	return fmt.Sprintf("byzantine %s: %d", e.Axis, e.Index)
 }
 
-// Repair attempts to repair an incomplete extended data
-// square (EDS), comparing repaired rows and columns against expected Merkle
-// roots.
-//
-// # Input
-//
-// Missing shares must be nil.
+// Repair attempts to repair an incomplete extended data square (EDS). The
+// parameters rowRoots and colRoots are the expected Merkle roots for each row
+// and column. rowRoots and colRoots are used to verify that a repaired row or
+// column is correct. Prior to the repair process, if a row or column is already
```

Comment:
Q: you can technically repair an EDS without the rowRoots and colRoots right?

Reply:
Yes it is possible because the underlying library (klauspost/reedsolomon) doesn't have a notion of verifying the repaired data against a row root or col root.